schema2Markdown.js

```diff
@@ -0,0 +1,254 @@
+const fse = require('fs-extra');
+const yaml = require('js-yaml');
+const path = require('path');
+
+function processProperty(propertyName, propertyDetails, schemaExamples) {
+    let markdownContent = '';
+
+    if (propertyName === 'type') {
+        markdownContent += `### Type\n\n`;
+        markdownContent += `\`${propertyDetails.const}\`\n\n`;
+    } else {
```

Comment:
I think you could use flow control to remove the `else` block (reducing a nesting layer), if you'd like. I think something like this would be easier to understand/maintain, but it's a personal preference.

(Not saying I'm opposed to `else` blocks. I just think *this* one in particular is unnecessary.)

```js
if (propertyName === 'type') {
    markdownContent += `### Type\n\n`;
    markdownContent += `\`${propertyDetails.const}\`\n\n`;
    return markdownContent;
}

markdownContent += `### ${propertyDetails.title || propertyName}\n\n`;
markdownContent += `\`${propertyName}\`\n\n`;
// rest of the function
```